test/integration/edit_proj_desc_test.rb

```diff
@@ -0,0 +1,102 @@
+require 'test_helper'
+
+class EditProjDescTest < ActionDispatch::IntegrationTest
+  include CapyHelper
+
+  self.use_transactional_fixtures = false
+
+  setup do
+    @project = projects(:description_test)
+    Capybara.current_driver = :webkit
+    Capybara.default_wait_time = 15
+  end
+
+  teardown do
+    finish
+  end
+
```

Comment:
Function has been def'ed.  Diff not suppressed because was not addressed at this very line.